multichain-testing/test/stake-ica.test.ts

```diff
@@ -180,7 +180,7 @@ const stakeScenario = test.macro(async (t, scenario: StakeIcaScenario) => {
     `rewards available on ${scenario.chain}`,
     {
       retryIntervalMs: 5000,
-      maxRetries: 8,
+      maxRetries: 18,
```

Comment:
consider documenting how this number was determined, for anyone in the future trying to adjust it. In particular, how can someone determine whether to lower it? What is the cost of raising it higher?

consider also making it a constant near the top of the file for easier discovery